src/functionaltests/resources/features/status.feature

```diff
@@ -0,0 +1,19 @@
+
+@Status
+Feature: Test Status
+
+  @TestStatusIsOk
+  Scenario: Test Status Is AVAILABLE
+    Given "None" services is unavailable
+    When I fetches status
```

Comment:
fetches --> fetch. But maybe we should use get as verb here, to be consistent? Change in all scenarios.

Reply:
Done.